infrastructure/io/src/test/java/tech/pegasys/teku/infrastructure/io/resource/FileResourceLoaderTest.java

```diff
@@ -43,10 +43,7 @@ public void shouldReturnEmptyWhenFileDoesNotExist(@TempDir Path tempDir) throws
   }
 
   @Test
-  public void shouldCreateInputStreamWhenPathIsADirectory(@TempDir Path tempDir) throws Exception {
-    // We could potentially return empty for directories, but it is going to be confusing for users
-    // to say we couldn't find something that exists. We should report that we couldn't read it.
-    assertThat(loader.load(tempDir.toAbsolutePath().toString())).isNotEmpty();
```

Comment:
Probably worth keeping these Windows test fixes to the other PR but that one's ready to merge anyway so should all work out.